packages/application-shell/src/components/fetch-user/fetch-user.mc.graphql

```diff
@@ -1,5 +1,3 @@
-#import "../project-switcher/project-switcher-fragments.mc.graphql"
```

Comment:
I had to remove this as the codegen was complaining of "duplicate types" related to `FetchLoggedInUser`